### PR TITLE
Fusetools2 572 clean code before starting

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: src/test/suite/z_extension.test.ts [hunk 1]
```diff
@@ -26,80 +26,60 @@ import { ACTIVATION_TIMEOUT } from './Utils';
...
 	}).timeout(ACTIVATION_TIMEOUT + 1000);	
 
-	test('test optional namespace support', function(done) {
+	test('test no namespace provided to getBaseCmd doesn\'t contain --namespace parameter', function() {
 		let cmdStrNoNS : string = kamel.getBaseCmd(`fakepath`,`fakecommand`, undefined);
 		assert.equal(cmdStrNoNS.indexOf('--namespace'), -1);
-
+	});
+	
+	test('test namespace provided to getBaseCmd contain --namespace parameter', function() {
```
Comment (bfitzpat) [inline src/test/suite/z_extension.test.ts:42]: contains instead of contain here

File: src/test/suite/z_extension.test.ts [hunk 2]
```diff
@@ -26,80 +26,60 @@ import { ACTIVATION_TIMEOUT } from './Utils';
...
-		// get NS from config settings
 		const testNs = 'testing';
-
-		// override namespace
 		await config.addNamespaceToConfig(testNs);
 
-		// re-retrieve namespace, should be test NS we specified
-		const resetNs : string | undefined = config.getNamespaceconfig();
+		const namespaceConfigValueAfterReset : string | undefined = config.getNamespaceconfig();
+		assert.equal(namespaceConfigValueAfterReset, testNs, 'The setted value should be available.');
```
Comment (bfitzpat) [inline src/test/suite/z_extension.test.ts:63]: "setted" is not really a word -- perhaps change to "The value set should be available"

**Review by bfitzpat — CHANGES_REQUESTED** [review]

File: src/test/suite/z_extension.test.ts [hunk 3]
```diff
@@ -26,80 +26,60 @@ import { ACTIVATION_TIMEOUT } from './Utils';
...
 		assert.ok(vscode.extensions.getExtension(extensionId));
-		done();
 	});
 
 	test('vscode-camelk extension should activate', async() => {
 		await utils.ensureExtensionActivated();
 	}).timeout(ACTIVATION_TIMEOUT + 1000);	
 
-	test('test optional namespace support', function(done) {
+	test('test no namespace provided to getBaseCmd doesn\'t contain --namespace parameter', function() {
```
[the remark this answers is not in the document]
Comment (apupier) [reply src/test/suite/z_extension.test.ts:37]: isn't "missing" to be used when something should be here but it isn't?
the --namespace is not really missing. it is not provided on purpose.

[the remark this answers is not in the document]
Comment (bfitzpat) [reply src/test/suite/z_extension.test.ts]: then I would say 'test that getBaseCmd doesn't contain --namespace parameter when no namespace is passed' - that's much more clear

**Review by bfitzpat — APPROVED** [review]

**Pull request merged**